src/components/note/__internal__/note.style.js

```diff
@@ -45,42 +45,37 @@ const StyledTitle = styled.header`
 `;
 
 const StyledFooterContent = styled.div`
-  position: relative;
   line-height: 21px;
-
-  &:first-of-type {
-    font-weight: bold;
-    font-size: 14px;
-    top: 16px;
-    height: 21px;
-  }
-
   ${({ theme }) => `
+    &:first-of-type {
+      font-weight: bold;
+      font-size: 14px;
+      margin-top: ${2 * theme.spacing}px;
+    }
+
     &:nth-of-type(2) {
       font-weight: bold;
       font-size: 12px;
       color: ${theme.note.timeStamp};
-      left: 16px;
-      top: 17px;
-      height: 21px;
+      margin-left: ${2 * theme.spacing}px;
+      margin-top: ${1 + (2 * theme.spacing)}px;
```

Comment:
Why is the extra 1px being added to the spacing multiplier here? Unless UX specifically want it as 17px I would argue for just using `2 * theme.spacing` so we can be consistent with using the spacing multiplier. 

On the other hand if it just needs to be 17px no matter what, I don't think you should use the spacing multiplier here

Reply:
The alignment is off, I assume because the font size is smaller for the last to content blocks. I agree I'll go with hardcoding

Reply:
Ah I see, just pulled it locally and had a look. You can leave this as 16px, all you need to do is set `align-items: baseline` on the div surrounding the content blocks, it's already a flex container so that will align the text correctly